March 2023 Updates (#1320)
akeneo/pim-helpcenter: content/updates/2023-03/01-Identifier Generator.md

---
pim_announcement_img: ./img/IDGenerator.png
pim_announcement_alt_img: Automate the creation of product identifiers in Akeneo PIM
pim_announcement_audience:
- EE
- GE
---

# Identifier Generator
::: meta-data type="New" features="Productivity" available="March" in="EE,GE" link-to-doc="https://help.akeneo.com/serenity-your-first-steps-with-akeneo/generate-product-identifiers?from_search=113993232"

> The product identifier is a critical piece of product information but often teams have to wait for the identifier to be created in another system such as the ERP or spreadsheet before creating the product in Akeneo PIM. Our new Identifier Generator enables product managers to automate the generation of product identifiers directly in Akeneo PIM and more efficiently prepare products for market by creating the product identifier in the same system where your product information lives. 

Reviewer: ```suggestion
The product identifier is a critical piece of product information but often teams have to wait for the identifier to be created in another system such as the ERP before enriching the product in Akeneo PIM. Our new Identifier Generator enables product managers to automate the generation of product identifiers directly in Akeneo PIM.
```
